File: Codeforces/231A.py
numberProblem = int(input())
write = 0

for i in range(numberProblem):
    count = 0
    
    petya,vasya,tonya = map(int, input().split())
    
    if petya == 1: count+=1
    if vasya == 1: count+=1
    if tonya == 1: count+=1

    if(count > 1): write+=1

print(write)

File: Codeforces/231A.kt
fun main(){

    val n = readLine()!!.toInt()
    var write = 0

    repeat(n){
        val (p,v,t) = readLine()!!.split(" ").map { it.toInt()}
        
        var count = 0
        if(p==1) count++
        if(v==1) count++
        if(t==1) count++

        if(count > 1) write++
    }
    print(write)
}



